Tiltfile

```diff
@@ -11,7 +11,7 @@ settings = {
     "kind_cluster_name": "capz",
     "capi_version": "v0.3.3",
     "cert_manager_version": "v0.11.0",
-    "feature_gates": '--feature-gates MachinePool=true'
+    "feature_gates": '--feature-gates=MachinePool=true'
```

Comment:
do we need to make this not true by default before merging?

Reply:
For development, I lean toward enabling experimental features by default. This opinion is not strongly held.

Reply:
Hmm not sure we should (I'll think about it some more) but if we do let's call that out in Tilt instructions in docs.

Reply:
👍